content/authoring/guidelines/coding.md

```diff
@@ -7,4 +7,23 @@ next: /authoring/guidelines/sample-tests/
 
 # General Kata Coding Guidelines
 
-_This page is a stub._
+Coding guidelines are usually a very big topic for programmers. Programmers like them, and programmers hate them. Programmers fight for them, and they fight against them. Coding guidelines can be very helpful, but can be also very difficult to define, establish, follow, and enforce.
+
+Kata are 90% code [citation needed]. But this code is not only compiled, run, and used to verify the correctness of user solutions by the automatic runner. Just like every other piece of code, it also needs to be debugged, fixed, updated, and maintained. Even if you write the most perfect code in the world, sooner or later it will be read by someone. Someone will want to learn from it, or someone will want to update it when some incompatible changes are introduced to the runner. Or there will be a bug which has to be handled. But even if your code is written perfectly and has no bugs, sooner or later it will have to be maintained in some way. And it's your responsibility to make it as easy as possible. Otherwise, users won't want to fix the kata and improve its quality, because no one wants to work with bad code.
```

Comment:
This definitely communicates the point but might be a bit heavy-handed. I'd recommend a direct and less proselytizing tone:

```suggestion
Kata code is not only used to verify the correctness of user solutions by the code runner; it also needs to be usable, understandable and maintainable by other users in the Codewars community. As such, all code in a kata, whether visible or invisible to the user, should adhere to a high standard of quality to ensure it can support these goals.
```